handler.go

```diff
@@ -411,7 +422,10 @@ MainLoop:
 		case command := <-handler.commandChan:
 			t1 := time.Now()
 			err = command.Execute(handler)
-			handler.Log.Fine("%s command %v took %s", handler, command, time.Since(t1))
+			handler.Log().WithFields(logrus.Fields{
+				"command": command,
+				"time": time.Since(t1),
+			}).Debug("handler command")
```

Comment:
```suggestion
			handler.Log().WithFields(logrus.Fields{
				"command": command,
				"time": time.Since(t1),
			}).WithError(err).Debug("Executed command")
```
Wiem, że niżej jest logowany błąd, ale tak naprawdę to nie wiadomo skąd ten błąd pochodzi (z którego switch case)

Reply:
Jak dla mnie to on w ogóle nie jest logowany niżej, tylko jest wrap `fmt.Errorf` (476) i zwracany na koniec funkcji.
Jeżeli jest jakikolwiek nie obsłużony błąd w tej pętli to cały handler rozłącza - to jest by design